assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserDimensionsPieChart.js

```diff
@@ -111,9 +148,11 @@ export default function UserDimensionsPieChart( {
 					'click',
 					onTooltipClick
 				);
+				currentContainerRef.removeEventListener( 'click', onExitClick );
+				currentContainerRef.removeEventListener( 'keyup', onEscape );
```

Comment:
We need to remove event listeners from the same object where they have been added:
```suggestion
				global.removeEventListener( 'click', onExitClick );
				global.removeEventListener( 'keyup', onEscape );
```